src/core/request/headers/index.ts

```diff
@@ -0,0 +1,143 @@
+import type {
+
+	BasicHeadersInit,
+	HeaderName,
+	HeaderValue,
+	NormalizedHeaderName,
+	NormalizedHeaderValue
+
+} from 'core/request/headers/interface';
+
+export * from 'core/request/headers/const';
+export * from 'core/request/headers/interface';
+
+export default class Headers {
+	protected readonly headers: Map<NormalizedHeaderName, NormalizedHeaderValue>;
+
+	constructor(init?: BasicHeadersInit) {
+		this.headers = new Map();
+
+		if (init instanceof Headers) {
+			for (const [name, value] of init.entries()) {
+				this.set(name, value);
+			}
+		} else if (Array.isArray(init)) {
```

Comment:
После каждого такого блока должен быть перевод строки

```suggestion

		} else if (Array.isArray(init)) {
```

Reply:
```
  iter = init.entries();
} else if (Array.isArray(init)) {
```
 а так нормально? если там одна строка